samples/WebApplication/Program.cs

```diff
@@ -26,7 +26,7 @@
     // the following extension method is available if you specify `GenerateSwashbuckleMappingExtensionMethod` - as shown above
     
     opt.MapVogenTypesInWebApplication();
-    opt.MapVogenTypesInWebApplicationShared();
```

Comment:
Previously, if the project named contained dots, then these would be stripped out, but now they're replaced with `_`. This was part of a bug fix for something else that derived the namespace from the project but failed where the project contained hyphens.